### ISSUE TITLE
Bug .lproj is broken if Base.lproj is missing

### ISSUE THREAD

**Issue by jerrymarino** [yonaskolb/XcodeGen#143]
I'm using XcodeGen for a large iOS app, which contains several dependencies and localizations. Many of these dependencies don't have a `Base.lproj` dir. XcodeGen will not handle the lproj's correctly if this is missing

**Comment by yonaskolb** [comment]
Thanks @jerrymarino. This is mentioned in #70.
@ryohey Could you summarise what changes we need to fix this?

**Comment by ryohey** [comment]
@jerrymarino @yonaskolb  We can't handle projects correctly that do not use Base Localization for now.
To fix that, we should change to specify the development region to not to use the base localization and fix some codes.
For now it may resolved by renaming the localization directory of your development language to Base.lproj.
I want to see the project to accurately reproduce the problem, Could you share your project?

**Comment by yonaskolb** [comment]
So if we basically just add up all the localizations we find and use them to set the known regions, and then have an editable development region we should be good for all cases

**Comment by ryohey** [comment]
Yes, I think that's right. It is good to add `developmentRegion` to `options`. 

```yaml
options:
  developmentRegion: ja # or Japanese, en, English and others defined in NSLocale should be supported
```

But this is per project preference. I don't know about future spec that may supports multiple projects #124) 
@yonaskolb How do you think about that?

**Comment by yonaskolb** [comment]
Yeah sounds good. 
Would anyone have time to submit a PR?

**Comment by ryohey** [comment]
Thanks I will fix this.

**Comment by jerrymarino** [comment]
@yonaskolb @ryohey awesome, thanks so much! I think this would make it a lot easier to use  👍  

It was confusing to me that it didn't `just work`. After I found it needed `Base.lproj` out, I ended up creating the `Base.lproj` for app, and in all of their dependencies that are localized.

**Comment by yonaskolb** [comment]
Once this is done, I'll cut a release

**Comment by yonaskolb** [comment]
Resolved in #155

**Comment by yonaskolb** [comment]
Actually #155 won't fix the issue of not having a `Base.lproj`
